report/renderer/dom.js

```diff
@@ -102,36 +105,23 @@ export class DOM {
   }
 
   /**
-   * @param {string} selector
-   * @param {ParentNode} context
-   * @return {!DocumentFragment} A clone of the template content.
-   * @throws {Error}
+   * @param {Parameters<typeof createComponent>[1]} componentName
+   * @return {!DocumentFragment} A clone of the cached component.
    */
-  cloneTemplate(selector, context) {
-    const template = /** @type {?HTMLTemplateElement} */ (context.querySelector(selector));
```

Comment:
for some reason I remembered the `templateContext` being more involved than this, but if this is all, what a nice thing to get to cut out :)